schema/schema_files.py

```diff
@@ -15,13 +14,20 @@
 TEST_FILE_TO_TEST_TYPE_MAP = {
     'collation_test': 'collation_short',
     'datetime_fmt_test_file': 'datetime_fmt',
+    'datetime_fmt_test': 'datetime_fmt',
+    'lang_name_test_file': 'lang_names',
     'lang_names_test_file': 'lang_names',
     'likely_subtags_test': 'likely_subtags',
     'list_fmt_test_file': 'list_fmt',
+    'list_fmt_test': 'list_fmt',
     'message_fmt2_test_file': 'message_fmt2',
+    'message_fmt2_test': 'message_fmt2',
     'num_fmt_test_file': 'number_fmt',
     'plural_rules_test_file': 'plural_rules',
-    'rdt_fmt_test_file': 'rdt'
+    'plural_rules_test': 'plural_rules',
+    'rdt_fmt_test_file': 'rdt',
+    'rdt_fmt_test_file': 'rdt',
```

Comment:
These two lines appear to be duplicates of each other

Reply:
Fixed, I think